lib/jasmine/runners/chromeheadless_boot.js

```diff
@@ -1,14 +1,56 @@
+/**
+ * Iterates the keys and values of an object.  Object.keys is used to extract the keys.
+ * @param object The object to iterate
+ * @param fn (value,key)=>{}
+ */
+function objectForEach(object, fn) {
+  Object.keys(object).forEach(key => {
+    fn(object[key],key, object)
+  })
+}
+
+function remove_jquery_elementes(object) {
+  objectForEach(object, (value, key, obj) => {
+    if (value instanceof jQuery) {
```

Comment:
This will throw an error if `jQuery` isn't in use on the page. We should make sure it actually exists before we do this check. Alternatively, we could do some cycle detection similar to https://github.com/jasmine/jasmine_selenium_runner/blob/master/lib/jasmine/runners/selenium.rb#L87

Reply:
I thougnt I look into what happes if `JQuery` is not defined but apperently i did not. I will find a way to check that `JQuery` is defined before doing this check.

Something along the line.
```suggestion
    if (typeof jQuery != "function" && value instanceof jQuery) {
```

This is not a cycle thing. th problem here it that is doesn't know how to cast a jQuery element to json. I guess it was to do with that the Jquery element has some refrences to the document lever dome nodes. I dont realy care too much what's causing it ¯\_(ツ)_/¯ I just want to run my spec without errormessages from gems.